### PR TITLE
Update newpkg.md

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Review by schillic — CHANGES_REQUESTED** [review]
Let me first check which of the two fixes does the trick.

**Comment by mforets** [comment]
ok sure. wanted to PR this so we dont forget. we also have to update JuliaReachTemplatePkg (which is anyways outdated).

**Comment by schillic** [comment]
This is wrong. What helped was removing `Coveralls` from the command chain.

File: docs/src/newpkg.md [hunk 1]
```diff
@@ -71,3 +71,13 @@ julia> using Pkg
...
 UUID("b4f0291d-fe17-52bc-9479-3d1a343d9043")
 ```
+
+## Adding code coverage
+
+Enable [codecov](https://codecov.io/) in the `.travis.yml` file:
+
+```julia
+codecov: true
+language: julia
```
Comment (schillic) [inline docs/src/newpkg.md:81]: It was said that the command should come *after* `language`. I do not think that this makes a difference, but who knows.

**Review by schillic — COMMENTED** [review]
Supposedly this does work. But I think we should first verify.

**Review by schillic — APPROVED** [review]

**Pull request merged**